substrate/bin/utils/chain-spec-builder/README.md

```diff
@@ -0,0 +1,117 @@
+# Chain Spec Builder
+
+Substrate's chain spec builder utility.
+
+A chain-spec is short for `chain-specification`. See the [sc-chain-spec](https://crates.io/crates/sc-chain-spec)
```

Comment:
Referring to docs make more sense:
```suggestion
A chain-spec is short for `chain-specification`. See the [sc-chain-spec](https://docs.rs/sc-chain-spec/latest/sc_chain_spec/)
```

Reply:
[37e43c0](https://github.com/paritytech/polkadot-sdk/pull/5954/commits/37e43c03090ee56e14c5df4da2c42945e6bcc08e)